python/ray/worker.py

```diff
@@ -831,11 +832,6 @@ def _process_task(self, task, function_execution_info):
             with profiling.profile("task:deserialize_arguments", worker=self):
                 arguments = self._get_arguments_for_execution(
                     function_name, args)
-        except RayTaskError as e:
```

Comment:
Why is this (or the equivalent with `RayError` instead) no longer needed?

Reply:
The below except block `except Exception as e` should also cover `RayError`